### DISCUSSION TITLE
Teacher account copilot not working

### DISCUSSION THREAD

**Discussion by dhust** [community/community#19230] [Copilot Conversations] [answered]
I've had a verified teacher account since January 2021. I've read that copilot is available for teachers/faculty. Is there something I need to do beyond having a teacher account? I had the preview for a few months. I've signed out and restarted VS Code but no luck.

**Comment by melvincwng** [comment]
@dhust, can you check your Copilot settings from the following steps:

1. 'Settings' > 'Copilot'
2. 'Settings' > 'Billing'

Would be useful if you could provide more information on those.

If you are eligible to use GitHub Copilot for free, you should see it clearly stated over there. Or else, you probably would need to pay for a subscription. 

As far as I'm aware of from [here](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/), GitHub Copilot is free to use for verified students and maintainers of popular open source software. The rest would instead get a 60 day free trial, after which will cost $10 USD/month or $100 USD/year.

Hope this clarifies.

**Comment by mashirozx** [comment]
<img width="821" alt="image" src="https://user-images.githubusercontent.com/16148054/175195601-39c10007-e622-4bf0-8497-1d1b45539a59.png">
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/16148054/175195644-1613a9e2-78e6-4b51-bb44-919f0db9575c.png">
<img width="1153" alt="image" src="https://user-images.githubusercontent.com/16148054/175195674-62d002f6-9977-4e8e-b812-46c1b1315ee3.png">

@melvincwng doesn't work on VSCode, copilot plugin v1.30.6165

**Reply by melvincwng** [reply]
@mashirozx, please contact [Github Support](https://support.github.com/contact/account) to resolve this issue.

**Comment by dhust** [comment]
It doesn't state that I'm eligible to use copilot for free, but I thought it came with a verified teacher account.
________________________________

**Reply by melvincwng** [reply]
@dhust, as far as I'm aware of from [here](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/), GitHub Copilot is free to use for verified students and maintainers of popular open source software. 

The rest would instead get a 60 day free trial, after which will cost $10 USD/month or $100 USD/year.

**Reply by melvincwng** [reply]
@dhust, if this reply has helped you, please kindly 'Mark as answer' so that others are aware that this issue has been addressed.

Thank you and have a nice day ahead 😄

**Comment by dhust** [comment]
I read that but it doesn't make sense to allow students to use copilot but not the teachers. Why would a teacher give students the ability to use it without knowing and experiencing everything it can do?
________________________________

**Reply by hsuanyi-chou** [reply]
Agree. Copilot should be free for teacher.

**Comment by eladcandroid** [comment]
I would like to express my support for a free teacher license, as a teacher for fullstack developers students here in Israel.

If you appreciate the importance of contributing to open source and future developers, the contribution of their teachers is surely no less.

**Comment by iromtal** [comment]
We frequently write live code during lecture. We want to share the same development environment as students. If we recommend it for class we also need to use it. Writing from MIT.

**Comment by joseluisgs** [comment, marked as answer]
https://github.com/community/community/discussions/19837
